addon/v1alpha1/types_managedclusteraddon.go

```diff
@@ -29,8 +29,68 @@ type ManagedClusterAddOn struct {
 	Status ManagedClusterAddOnStatus `json:"status"`
 }
 
-// ManagedClusterAddOnSpec is empty for now.
+// ManagedClusterAddOnSpec defines the install configuration of
+// an addon agent on managed cluster.
 type ManagedClusterAddOnSpec struct {
+	// installNamespace is the namespace on the managed cluster to install the addon agent.
+	// If it is not set, open-cluster-management-addon namespace is used to install the addon agent.
```

Comment:
The default namespace may be `open-cluster-management-agent-addon`, ACM use `open-cluster-management-agent-addon` namespace as the addons namespace now